packages/api/src/domain/medical/__tests__/location-address.ts

```diff
@@ -1,12 +1,13 @@
 import { fakerEN_US as faker } from "@faker-js/faker";
 import { AddressStrict } from "../location-address";
+import { USState } from "@metriport/core/domain/geographic-locations";
 
 export const makeAddressStrict = (): AddressStrict => {
   return {
     addressLine1: faker.location.streetAddress(),
     zip: faker.location.zipCode(),
     city: faker.location.city(),
-    state: faker.location.state({ abbreviated: true }),
+    state: faker.location.state({ abbreviated: true }) as USState,
```

Comment:
Not to be nitpicky, but it is good that we got here. We could return a random element of the Enum values instead, this would prevent for example returning Hawaii if its not on the USState enum.